hack/verify-staging-imports.sh

```diff
@@ -31,4 +31,9 @@ for dep in $(ls -1 ${KUBE_ROOT}/staging/src/k8s.io/); do
 	fi
 done
 
+if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
```

Comment:
> oh, ic. No import paths anymore for staging. Do we really want that? This way one can go get via the github url and this will probably break our package.

This stops you from claiming your canonical package is in the kubernetes repo when its actually in apimachinery.